airflow/contrib/operators/aws_lambda_operator.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.contrib.hooks.aws_lambda_hook import AwsLambdaHook
+
+
+class AwsLambdaInvokeFunctionOperator(BaseOperator):
+    """
+    Invoke AWS Lambda functions with a JSON payload.
+
+    The check_success_function signature should be a single param which will receive a dict.
+    The dict will be the "Response Structure" described in http://bit.ly/2KXLWWF. It may
+    be necessary to read the Payload see the actual response from the Lambda function e.g.,
+
+    ```
+    def succeeded(response):
+        payload = json.loads(response['Payload'].read())
+        # do something with payload
+    ```
+
+    :param function_name: The name of the Lambda function.
+    :type function_name: str
+    :param region_name: AWS region e.g., eu-west-1, ap-southeast-1, etc.
+    :type region_name: str
+    :param payload: The JSON to submit as input to a Lambda function.
+    :type payload: str
+    :param check_success_function: A function to check the Lambda response and determine success or failure.
+    :type check_success_function: function
+    :param log_type: Tail Invocation Request
+    :type log_type: str
+    :param qualifer: A version or alias name for the Lambda.
```

Comment:
```suggestion
    :param qualifier: A version or alias name for the Lambda.
```
and below in `:type`